plugins/terraform/src/components/TerraformLatestRunContent/TerraformLatestRunContent.test.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  render,
+  screen,
+} from '@testing-library/react';
+import React from 'react';
+import { TerraformLatestRunContent } from './TerraformLatestRunContent';
+
+jest.mock('@backstage/core-components', () => {
+  const originalModule = jest.requireActual('@backstage/core-components');
+
+  const MockInfoCard = (props: React.PropsWithChildren<{ title: string }>) => (
+    <div>
+      <p>{`Mock InfoCard: ${props.title} `}</p>
+      {props.children}
+    </div>
+  )
+
+  const MockLogViewer = ({ text }: { text: string }) => (
+    <div>{`Mock TerraformRuns: ${text}`}</div>
+  );
+
+  return {
+    __esModule: true,
+    InfoCard: MockInfoCard,
+    LogViewer: MockLogViewer,
```

Comment:
Looks like `LogViewer` and `Progress` aren't actually used by the component, are the mocks required here?

Reply:
Removed.